source/edge-server/connect.txt

```diff
@@ -201,27 +227,82 @@ You can connect to the Edge Server using standard MongoDB Drivers and tools.
 Clients use a specialized MongoDB URI connection string to connect to 
 Edge Server and send requests.
 
-The wire protocol server is on by default. You can connect to it via the 
-host machine using this MongoDB URI:
+The Wire Protocol server accepts these incoming connections, and is on by 
+default. You can connect to it via the host machine using this MongoDB URI:
 
 .. code:: console
 
-   mongodb://EDGE_SERVER_HOSTNAME:27021
+   mongodb://<username>:<password>@<hostname>:<port>?authMechanism=PLAIN&appName=::<auth-type>
```

Comment:
I was torn about how to document this, since `appName` is (currently) completely optional and is actually ignored in the code path.

So this _could_ be:

`appName=<your-app-name>` or it could be omitted altogether. I punted and just went with the engineering example, but I'm open to your thoughts about how to make this clear in the example, or whether we should just omit it.

Reply:
this is correct. appName is optional and can be omitted if you're using username password. if you use other auth providers (not yet supported) you'll need that extra field.

worth also noting that you can start up the edge server with `--insecure-disable-auth`, which will allow you to connect without specifying a user/pass (as before), at which point you can do `mongodb://EDGE_SERVER_HOSTNAME:27021`.